packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -93,10 +94,15 @@ class VmServiceWrapper implements VmService {
         ));
   }
 
-  Future<Success> clearCpuProfile(String isolateId) async {
-    final response = await _trackFuture('clearCpuProfile',
-        callMethod('_clearCpuProfile', isolateId: isolateId));
-    return response as Success;
+  @override
+  Future<Success> clearCpuSamples(String isolateId) async {
+    if (await isProtocolVersionLessThan(major: 3, minor: 27)) {
+      final response = await _trackFuture('clearCpuProfile',
```

Comment:
nit: for consistency either make this `_trackFuture('_clearCpuProfile'` or `_trackFuture('clearCpuSamples``

Reply:
Done.